.github/semantic.yml

```diff
@@ -0,0 +1,4 @@
+titleOnly: false
+commitsOnly: false
+titleAndCommits: true
+allowMergeCommits: false
```

Comment:
We often use `true / false / false / true` here (see [analytics](https://github.com/dhis2/analytics/blob/master/.github/semantic.yml))

Reply:
changed